kfs-purap/src/main/java/edu/arizona/kfs/module/purap/document/web/struts/PayeeSearchAction.java

```diff
@@ -0,0 +1,95 @@
+package edu.arizona.kfs.module.purap.document.web.struts;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.apache.log4j.Logger;
+import org.apache.struts.action.ActionForm;
+import org.apache.struts.action.ActionForward;
+import org.apache.struts.action.ActionMapping;
+import org.kuali.kfs.sys.context.SpringContext;
+import org.kuali.rice.kim.api.role.Role;
+import org.kuali.rice.kim.api.role.RoleService;
+import org.kuali.rice.krad.util.GlobalVariables;
+import org.kuali.rice.kns.web.struts.action.KualiAction;
+import org.kuali.rice.kim.api.services.KimApiServiceLocator;
+import org.kuali.rice.krad.exception.AuthorizationException;
+import org.kuali.rice.krad.util.KRADConstants;
+import org.kuali.rice.kim.api.KimConstants;
+
+import edu.arizona.kfs.tax.TaxConstants;
+import edu.arizona.kfs.tax.businessobject.Payee;
+import edu.arizona.kfs.tax.businessobject.Payer;
+import edu.arizona.kfs.tax.document.web.struts.PayeeSearchForm;
+import edu.arizona.kfs.sys.KFSConstants;
+import edu.arizona.kfs.sys.KFSKeyConstants;
+import edu.arizona.kfs.module.purap.service.TaxReporting1099Service;
+
+public class PayeeSearchAction extends KualiAction {
+    private static final Logger LOG = Logger.getLogger(PayeeSearchAction.class);
+
+    private static String TOTAL_TAX_AMOUNT = "1099_TOTAL_TAX_AMOUNT";
+    private static String TAX_AMOUNT_BY_PAYMENT_TYPE = "1099_TAX_AMOUNT_BY_PAYMENT_TYPE";
+    private static String TAX_YEAR = "taxYear";
```

Comment:
I noticed that this value can be referenced at TaxPropertyConstants.TAX_YEAR. When you make the above mentioned change, you should reference that constant as well.